packages/sitecore-jss/src/utils/app-root-resolver.ts

```diff
@@ -0,0 +1,52 @@
+import { GraphQLRequestClient } from '../graphql-request-client';
+import { SitecoreTemplateId } from '../constants';
+
+const appRootQuery = /* GraphQL */ `
```

Comment:
IIRC, before we decided to use "site" because we are passing in site name to sitemap service in order to figure out the root item. But in this new resolver, it is searching for an item in the Sitecore tree that uses the the `App` template (/sitecore/templates/Foundation/JavaScript Services/App), so that's why I believe "app root" is more accurate.

Reply:
That's right, that was about the `siteName` parameter (vs `appName`). Makes sense as-is. I think I was also thrown off by the `getSiteRoot` query name: https://github.com/Sitecore/jss/blob/feature/469713/packages/sitecore-jss/src/context/resolve-app-root.ts#L5
Maybe change that to match?

Reply:
Agreed. In the next PR (sharing the app root query) I make sure to use "appRootId" everywhere. I prefer to go ahead and close this one out before adding that additional work on top.